packages/calypso-e2e/src/lib/pages/signup/user-signup-page.ts

```diff
@@ -154,10 +154,13 @@ export class UserSignupPage {
 	async signupWoo( email: string ): Promise< NewUserResponse > {
 		await this.page.fill( selectors.emailInput, email );
 
-		const [ , response ] = await Promise.all( [
-			this.page.waitForURL( /.*woocommerce\.com*/, { waitUntil: 'networkidle' } ),
-			this.page.waitForResponse( /.*new\?.*/ ),
+		const [ response ] = await Promise.all( [
+			this.page.waitForResponse( /.*users\/new\?.*/ ),
```

Comment:
More specific to the API URL to avoid potential future breakage due to `new` being too generic